maldoca/base/get_runfiles_dir.cc

```diff
@@ -118,7 +119,14 @@ absl::Status IsDirectory(const std::string& name) {
 
 std::string GetRunfilesDir() {
   std::string bin_path = GetExecutablePath();
-  std::string runfiles_suffix = ".runfiles/com_google_maldoca";
+  std::string sep;
+#if defined(_WIN32)
+  sep = base::WideToUTF8(
+      base::WStringPiece({base::FilePath::kSeparators[0], '\0'}));
```

Comment:
@david: do u know a better approach? (the compiler doesn't like a single char but wants a string due to the "&")

Reply:
filepath will accept / as separator for windows. Check the comment in mini_chromium/base/files/file_path.h

Reply:
Ok I am not sure what you want to do but instead of doing this you can use filepath's  append () to construct the constant runfile path and get it back as a string. Then it's platform independent since that's all take care of in the  filepath